float32 camera_pan
float32 camera_tilt
uint8 mode
uint8 POSITION_MODE=0
uint8 VELOCITY_MODE=1
